### ISSUE TITLE
node --harmony

### ISSUE THREAD

**Issue by kxuanCeltik** [humanmade/cf-to-tf#3]
$ cf-to-tf -h
module.js:549
    throw err;

Error: Cannot find module '/home/juan/node --harmony'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3

Please help, thanks.

**Comment by Manuscrit** [comment]
Similar problem here : 
(UNIX on Windows 10)

`cf-to-tf --stack dpm-ansible-dev config`

> /usr/bin/env: «node --harmony»: Aucun fichier ou dossier de ce type

**Comment by neoshadybeat** [comment]
same as above

**Comment by david1983** [comment]
same

**Comment by iAmAska** [comment]
I had a similar issue and fixed it by changing

`#!/usr/bin/env node --harmony`

to

`#!/usr/bin/env node`

in `index.js`

**Comment by nathanielks** [comment]
SO sorry for the silence here, folks! My email client was super buggy when this was filed and I never received a notification. This has been resolved by @zackse in  https://github.com/humanmade/cf-to-tf/pull/6

**Comment by marcodatola** [comment]
Hi, after removing `--harmony` from `../lib/node_modules/@humanmade/cf-to-tf/index.js`
when I run the command, I get the following error.......
```
[ec2-user@ip-10-204-3-229 bin]$ cf-to-tf ~/IoT-Analytics/Demo/examplestack.json config
/home/ec2-user/.nvm/versions/node/v4.4.5/lib/node_modules/@humanmade/cf-to-tf/lib/shared.js:4
                const { stringify = false, allowMissing = false } = opts;
                      ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/ec2-user/.nvm/versions/node/v4.4.5/lib/node_modules/@humanmade/cf-to-tf/lib/configuration.js:1:78)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

**Comment by nathanielks** [comment]
Hi @marcodatola! I can see you're using node version 4.4.5, which was EOL'd quite a bit of time ago. Can you verify it works on a more up to do date version, using at least Node 8.x?

**Comment by josegonzalez** [comment]
@nathanielks would it be possible for you to release the update to npm? It seems like the latest release there is 0.1.1 at the moment...

**Comment by nathanielks** [comment]
Apologies, @josegonzalez! I've went ahead and published 0.1.2 and 0.1.3 (which is a security patch release).

**Comment by josegonzalez** [comment]
<3 Thanks!
